### PR TITLE
feat(compute): Adding script for OS Login with physical keys

### PR DESCRIPTION
**Pull request by m-strzelczyk**
## Description

Fixes b/218664522

There is not test associated with this script, as it's impossible to automate a login procedure that requires physical security key. I have checked this script manually and it works.

## Checklist
- [x] I have followed [Sample Guidelines from AUTHORING_GUIDE.MD](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/AUTHORING_GUIDE.md)
- [x] README is updated to include [all relevant information](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/AUTHORING_GUIDE.md#readme-file)
- [x] **Tests** pass:   `nox -s py-3.6` (see [Test Environment Setup](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/AUTHORING_GUIDE.md#test-environment-setup))
- [x] **Lint** pass:   `nox -s lint` (see [Test Environment Setup](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/AUTHORING_GUIDE.md#test-environment-setup))
- [ ] These samples need a new **API enabled** in testing projects to pass (let us know which ones)
- [ ] These samples need a new/updated **env vars** in testing projects set to pass (let us know which ones)
- [x] Please **merge** this PR for me once it is approved.

### CODE REVIEW

File: compute/oslogin/physical_security_key_ssh.py [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+    command.append("{username}@{ip}".format(username=username, ip=ip_address))
+    return command
+
+
+def main(user_key, ip_address, dryrun, directory=None):
+    """Configure SSH key files and print SSH command."""
+    directory = directory or os.path.join(os.path.expanduser("~"), ".ssh")
+
+    # Create the OS Login API object.
+    oslogin = googleapiclient.discovery.build("oslogin", "v1beta")
```
[the remark this answers is not in the document]
Comment (m-strzelczyk) [reply compute/oslogin/physical_security_key_ssh.py:50]: Sounds like a good idea! I'll try it out.

[the remark this answers is not in the document]
Comment (m-strzelczyk) [reply compute/oslogin/physical_security_key_ssh.py:50]: Actually, it's not possible. The GAPIC client doesn't support beta API, and this code sample requires the beta API :<

**Comment by m-strzelczyk** [comment]
Can someone please review?

File: compute/oslogin/physical_security_key_ssh.py [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+    # Retrieve security keys and OS Login username from a user's Google account.
+    profile = (
+        oslogin.users()
+        .getLoginProfile(name="users/{}".format(user_key), view="SECURITY_KEY")
+        .execute()
+    )
+    security_keys = profile.get("securityKeys")
+
+    if "posixAccounts" not in profile:
+        print("You need to have at least one OS Login enabled VM in your project.")
```
Comment (illfelder) [inline compute/oslogin/physical_security_key_ssh.py:61]: If you don't have a posix account configured, that means the user hasn't configured a POSIX account. That doesn't have any relation to whether OS Login is enabled on a VM in the project.

File: compute/oslogin/physical_security_key_ssh.py [hunk 3]
```diff
@@ -0,0 +1,99 @@
+#!/usr/bin/env python
```
Comment (parthea) [inline compute/oslogin/physical_security_key_ssh.py:1]: Should there also be a test file? `compute/oslogin/physical_security_key_ssh_test.py`

**Comment by illfelder** [comment]
Looks good to me!

**Review by illfelder — APPROVED** [review]

**Review by leahecole — APPROVED** [review]
Acting as second stamp now that we have a product approver and @parthea 😁

**Pull request merged**